src/Integration.Vsix/SonarLint.vsct

```diff
@@ -20,12 +20,52 @@
   <Include href="Commands\PackageCommands.vsct"/>
   <Include href="SonarLintDaemon\DaemonCommands.vsct"/>
 
+  <Commands package="guidVsPackagePkgString">
+
+    <Groups>
+      <Group guid="guidCmdSet" id="SonarLintMenuGroup" priority="0x0600">
```

Comment:
This priority will affect where the `SonarLint` menu appears on the `Extensions` menu.
Does it appear where we want it to?